EIPS/eip-6963.md

```diff
@@ -296,3 +229,100 @@ The use of SVG images introduces a cross-site scripting risk as they can include
 ## Copyright
 
 Copyright and related rights waived via [CC0](../LICENSE.md).
+
+
+[^rfc4122]:
```

Comment:
@pedrouid do you care how we end up citing these RFCs? With #7113 incoming I think we can do this more easily now using the usual markdown linking normally used.